zkstack_cli/crates/common/src/server.rs

```diff
@@ -4,9 +4,16 @@ use xshell::{cmd, Shell};
 
 use crate::cmd::Cmd;
 
+/// Default command to run the server; will use `cargo` to build it.
+const DEFAULT_SERVER_COMMAND: &str =
+    "cargo run --manifest-path ./core/Cargo.toml --release --bin zksync_server";
+
 /// Allows to perform server operations.
 #[derive(Debug)]
 pub struct Server {
+    /// Command to run the server, could be a path to the binary.
+    /// If not set, the server will be built using cargo.
+    server_command: Option<String>,
```

Comment:
TBH, i'm a bit of afraid when we have such flexibility. doesn't look useful anywhere except probably ci